src/test/utils.ts

```diff
@@ -21,15 +21,16 @@ export const generateNextWithAssertions = (
 		runAssertions(err).then(done).catch(done);
 	});
 
-export const getTestUserAuthenticationId = () => 'foo@example.com';
+export const getTestUserKeycloakUserId = () =>
+	'00000000-0000-0000-0000-000000000001'; // This value is not a reference, it's just a static GUID
```

Comment:
This is extremely picky, so feel free to ignore me, but can this be a more obviously different value than the system ID?

```suggestion
	'11111111-1111-1111-1111-111111111111'; // This value is not a reference, it's just a static GUID
```

Reply:
I'm all here for this kind of picky -- good idea.